resolver/resolver.cc

```diff
@@ -737,8 +737,8 @@ class ResolveConstantsWalk {
         return true;
     }
 
-    static bool resolveConstants(const core::GlobalState &gs, vector<ResolveItems<ResolutionItem>> &jobs,
-                                 WorkerPool &workers) {
+    static bool resolveResolutionItems(const core::GlobalState &gs, vector<ResolveItems<ResolutionItem>> &jobs,
```

Comment:
What do you think about going further with this change:

1. rename `ResolutionItem` as `ConstantResolutionItem` for parallelism with all the other `*ResolutionItem` structures;
2. renaming this function `resolveConstantResolutionItems`